lhotse/custom.py

```diff
@@ -81,6 +82,9 @@ def __delattr__(self, key: str) -> None:
             raise AttributeError(f"No such member: '{key}'")
         del self.custom[key]
 
+    def to_dict(self) -> Dict[str, Any]:
+        return asdict(self)
```

Comment:
Can you replace it with `asdict_nonull` imported from `lhotse.utils`? Thanks

Reply:
Done